### ISSUE TITLE
Update for babel 6?

### ISSUE THREAD

**Issue by gnarf** [anvk/babel-npm-module-example#2]
Can you update this for the babel 6 workflow?

**Comment by anvk** [comment]
Sure. Give me some time, i'm making dinner right now...

**Comment by gnarf** [comment]
Was trying to set this up myself:

```
[16:03:07] Failed to load external module babel-core/register
[16:03:07] Failed to load external module babel/register
[redacted]/gulpfile.babel.js:1
(function (exports, require, module, __filename, __dirname) { import gulp from
                                                              ^^^^^^
SyntaxError: Unexpected reserved word
    at exports.runInThisContext (vm.js:73:16)
    at Module._compile (module.js:443:25)
    at Object.Module._extensions..js (module.js:478:10)
    at Module.load (module.js:355:32)
    at Function.Module._load (module.js:310:12)
    at Module.require (module.js:365:17)
    at require (module.js:384:17)
    at Liftoff.handleArguments (/usr/local/lib/node_modules/gulp/bin/gulp.js:116:3)
    at Liftoff.<anonymous> (/usr/local/lib/node_modules/gulp/node_modules/liftoff/index.js:192:16)
    at module.exports (/usr/local/lib/node_modules/gulp/node_modules/liftoff/node_modules/flagged-respawn/index.js:17:3)
```

Not quite sure what I'm missing.

**Comment by gnarf** [comment]
Oh hey - found it with `npm instal --save-dev babel-core`

**Comment by anvk** [comment]
Resolved in `0.0.6`. Please give it a try and let me know if it works for you.
